### PR TITLE
Prepare bam 21.04.1

### PR DESCRIPTION
**Pull request by nmagniezCentreon**
## Description

Release Note 21.04.1

## Target serie

- [ ] 20.04.x
- [ ] 20.10.x
- [x] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x (master)

### CODE REVIEW

File: en/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -58,6 +58,25 @@ If you have feature requests or want to report a bug, please contact support.
...
 ## Centreon BAM
 
+### 21.04.1
+
+`January 7, 2022`
+
+#### Improvements
+
+- Add Feature Flipping for Resources Status vs Legacy Pages
+- ACL: Authorize visualisation on daughter Business Activities even if not part of the same Business View
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:68]:
```suggestion
- ACL: Authorize visualisation on child Business Activities even if not part of the same Business View
```

File: en/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -58,6 +58,25 @@ If you have feature requests or want to report a bug, please contact support.
...
+- Add warning when enabling notifications for a BA, for the user to know about the necessity of setting notification commands
+
+#### Bugfixes
+
+- Fixed the message in popup when deleting "Activités Métiers" (Fr language)
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:73]: not totally clear

File: en/releases/centreon-commercial-extensions.md [hunk 3]
```diff
@@ -58,6 +58,25 @@ If you have feature requests or want to report a bug, please contact support.
...
+- Fixed an error when generating poller configuration with a disabled BA linked to an escalation task
+- Fixed the relations between BA and escalations are exported in RS engine conf regardless the BA is linked to the RS or not
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:75]: there are 2 verbs in the sentence

File: en/releases/centreon-commercial-extensions.md [hunk 4]
```diff
@@ -58,6 +58,25 @@ If you have feature requests or want to report a bug, please contact support.
...
+- Fixed the alignment of "required" input text on panel in Add Business Activity panel
+- Fixed MBI menu entry is not visible after upgrade from 19.10 to 21.10.0-beta.2 due to BAM module
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:77]: 2 verbs

File: en/releases/centreon-commercial-extensions.md [hunk 5]
```diff
@@ -58,6 +58,25 @@ If you have feature requests or want to report a bug, please contact support.
...
+- Add Feature Flipping for Resources Status vs Legacy Pages
+- ACL: Authorize visualisation on daughter Business Activities even if not part of the same Business View
+- Add warning when enabling notifications for a BA, for the user to know about the necessity of setting notification commands
+
+#### Bugfixes
+
+- Fixed the message in popup when deleting "Activités Métiers" (Fr language)
+- Fixed an error when generating poller configuration with a disabled BA linked to an escalation task
+- Fixed the relations between BA and escalations are exported in RS engine conf regardless the BA is linked to the RS or not
+- Fixed the alignment of "required" input text on panel in Add Business Activity panel
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:76]: in panel + in panel?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: en/releases/centreon-commercial-extensions.md [hunk 6]
```diff
@@ -58,6 +58,25 @@ If you have feature requests or want to report a bug, please contact support.
...
+
+#### Improvements
+
+- Add Feature Flipping for Resources Status vs Legacy Pages
+- ACL: Authorize visualisation on child Business Activities even if not part of the same Business View
+- Add warning when enabling notifications for a BA, for the user to know about the necessity of setting notification commands
+
+#### Bugfixes
+
+- Fixed the wrong message in popup when deleting "Activités Métiers" (Fr language)
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:73]:
```suggestion
- Fixed the message that appeared in a popup when deleting "Activités Métiers" (Fr language)
```

File: en/releases/centreon-commercial-extensions.md [hunk 7]
```diff
@@ -58,6 +58,25 @@ If you have feature requests or want to report a bug, please contact support.
...
+- Fixed an error when generating poller configuration with a disabled BA linked to an escalation task
+- Fixed the relations between BA and escalations exported in RS engine conf regardless if the BA is linked to the RS or not
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:75]:
```suggestion
- Fixed the relations between BA and escalations exported in RS engine conf regardless of whether the BA is linked to the RS or not
```

File: en/releases/centreon-commercial-extensions.md [hunk 8]
```diff
@@ -58,6 +58,25 @@ If you have feature requests or want to report a bug, please contact support.
...
+- Fixed the alignment of "required" input text on panel in Add Business Activity
+- Fixed MBI menu entry not visible after upgrade from 19.10 to 21.10.0-beta.2 due to BAM module
```
Comment (cg-tw) [inline en/releases/centreon-commercial-extensions.md:77]:
```suggestion
- Fixed MBI menu entry that was not visible after upgrade from 19.10 to 21.10.0-beta.2 due to BAM module
```

File: fr/releases/centreon-commercial-extensions.md [hunk 9]
```diff
@@ -59,6 +59,25 @@ commerciales, veuillez contacter le support.
...
 ## Centreon BAM
 
+### 21.04.1
+
+`January 7, 2022`
+
+#### Improvements
+
+- Add Feature Flipping for Resources Status vs Legacy Pages
+- ACL: Authorize visualisation on daughter Business Activities even if not part of the same Business View
```
Comment (cg-tw) [inline fr/releases/centreon-commercial-extensions.md:69]:
```suggestion
- ACL: Authorize visualisation on child Business Activities even if not part of the same Business View
```

File: fr/releases/centreon-commercial-extensions.md [hunk 10]
```diff
@@ -59,6 +59,25 @@ commerciales, veuillez contacter le support.
...
+- Add warning when enabling notifications for a BA, for the user to know about the necessity of setting notification commands
+
+#### Bugfixes
+
+- Fixed the wrong message in popup when deleting "Activités Métiers" (Fr language)
```
Comment (cg-tw) [inline fr/releases/centreon-commercial-extensions.md:74]:
```suggestion
- Fixed the message that appeared in a popup when deleting "Activités Métiers" (Fr language)
```

File: en/releases/centreon-commercial-extensions.md [hunk 11]
```diff
@@ -58,6 +58,25 @@ If you have feature requests or want to report a bug, please contact support.
 
 ## Centreon BAM
 
+### 21.04.1
+
+`January 7, 2022`
```
Comment (tuntoja) [inline en/releases/centreon-commercial-extensions.md:63]:
```suggestion
`January 21, 2022`
```

File: fr/releases/centreon-commercial-extensions.md [hunk 12]
```diff
@@ -59,6 +59,25 @@ commerciales, veuillez contacter le support.
...
+- Fixed an error when generating poller configuration with a disabled BA linked to an escalation task
+- Fixed the relations between BA and escalations exported in RS engine conf regardless the BA is linked to the RS or not
```
Comment (cg-tw) [inline fr/releases/centreon-commercial-extensions.md:76]:
```suggestion
- Fixed the relations between BA and escalations exported in RS engine conf regardless of whether the BA is linked to the RS or not
```

File: fr/releases/centreon-commercial-extensions.md [hunk 13]
```diff
@@ -59,6 +59,25 @@ commerciales, veuillez contacter le support.
 
 ## Centreon BAM
 
+### 21.04.1
+
+`January 7, 2022`
```
Comment (tuntoja) [inline fr/releases/centreon-commercial-extensions.md:64]:
```suggestion
`January 21, 2022`
```

File: fr/releases/centreon-commercial-extensions.md [hunk 14]
```diff
@@ -59,6 +59,25 @@ commerciales, veuillez contacter le support.
...
+- ACL: Authorize visualisation on daughter Business Activities even if not part of the same Business View
+- Add warning when enabling notifications for a BA, for the user to know about the necessity of setting notification commands
+
+#### Bugfixes
+
+- Fixed the wrong message in popup when deleting "Activités Métiers" (Fr language)
+- Fixed an error when generating poller configuration with a disabled BA linked to an escalation task
+- Fixed the relations between BA and escalations exported in RS engine conf regardless the BA is linked to the RS or not
+- Fixed the alignment of "required" input text on panel in Add Business Activity
+- Fixed MBI menu entry not visible after upgrade from 19.10 to 21.10.0-beta.2 due to BAM module
```
Comment (cg-tw) [inline fr/releases/centreon-commercial-extensions.md:78]:
```suggestion
- Fixed MBI menu entry that was not visible after upgrade from 19.10 to 21.10.0-beta.2 due to BAM module
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**